diff --git src/faust/fsynth.lib src/faust/fsynth.lib
index 8f29394..8d9c62c 100644
--- src/faust/fsynth.lib
+++ src/faust/fsynth.lib
@@ -104,7 +104,7 @@ fmosc(o1f,o1sr,o1st,o2f,o2d,na,wa,ta,adsron) = freqset : fmosc
     o1sr1 = 2 : pow(o1sr) : ba.sAndH(ta : ba.impulsify);
     o1st1 = o1st : ba.sAndH(ta : ba.impulsify);
     ta1 = ta;
-    cf = o1f : +(o1f : *(en.ar(0,o1st1,ta1) : pow(2.7) : *(o1sr1))) : min(22050); //calculate op1 freq during freq slide
+    cf = o1f : +(o1f : *(en.ar(0,o1st1,ta1) : pow27 : *(o1sr1))) : min(22050); //calculate op1 freq during freq slide
     freqcontrol =  cf : +(na) : +(os.oscsin(cf*o2f) : *(cf) : *(o2d)); //where the freq modulation happens
     freqset = _ : *(freqcontrol) : +(freqcontrol);
     clip(x) = x : min(1,_) : max(0,_);
